src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
+      <div class="calendar__field"></div>
```

Comment:
You have 31 div elements with the class 'calendar__field'. However, according to the task requirements, each of these divs should contain a number from 1 to 31. This number represents the day of the month. So, you should add a number inside each div. For example, the first div should contain the number 1, the second div should contain the number 2, and so on up to the number 31 in the last div.